src/message/cachedMessageRender.js

```diff
@@ -21,9 +21,12 @@ export default (
   if (!isEqual(lastRenderedMessages, renderedMessages)) {
     const rendered: Object[] = renderedMessages.reduce((result, section) => {
       result.push(
-        <MessageListSection key={section.key} message={section.message} />,
+        Object.keys(section.message).length > 0 && (
+          <MessageListSection key={section.key} message={section.message} />
```

Comment:
Move the check in the `MessageListSection` object.
Change the `if (!message) return null;` to include the `Object.keys(message).length` too